Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "3.0.3"
-gem "pg"
```

Comment:
Why did you remove the PostgreSQL gem? You will occupy it when you deploy to Heroku.

Reply:
Before this we implemented the gem 2 times we needed to delete one in order to avoid conflicts